standards/views.py

```diff
@@ -48,15 +48,18 @@ def by_curriculum_csv(request, slug):
 
     writer = csv.writer(response, encoding='utf-8')
     writer.writerow(['curriculum', 'unit', 'lesson #', 'lesson name',
-                     'standard framework', 'standard', 'cross curricular opportunity'])
+                     'standard framework', 'standard', 'category', 'description', 'cross curricular opportunity'])
     for unit in curriculum.units:
         for lesson in unit.lessons:
             for standard in lesson.standards.all():
                 writer.writerow([curriculum.slug, unit.slug, 'lesson %d' % lesson.number, lesson.title,
-                                 standard.framework.slug, standard.shortcode, False])
+                                 standard.framework.slug,
+                                 standard.shortcode,
+                                 standard.category.name, standard.name, False])
```

Comment:
I'm not super familiar with python syntax, but this indentation looks weird to me. Can we either give each item its own line, or group them into several lines of similar length?